src/Rector/New_/StringToArrayArgumentProcessRector.php

```diff
@@ -96,6 +102,22 @@ private function processArgumentPosition(New_|MethodCall $node, int $argumentPos
             return null;
         }
 
+        if ($node instanceof MethodCall) {
```

Comment:
AST resolver is quite heavy to handle this case.
There should be check for allowed method names, that can be converted.

Reply:
implemented https://github.com/rectorphp/rector-symfony/pull/70/commits/67323a8e664834a510efe4098eddeb92a744a299